### DISCUSSION TITLE
Modyfikacja zasobu GET /order/customer-returns

### DISCUSSION THREAD

**Discussion by martacronic** [allegro/allegro-api#5669] [Suggestions - resolved]
![image](https://user-images.githubusercontent.com/37405655/167612545-deb29e8a-0bb1-41c1-8327-b941d565f0bb.png)

W tej chwili na stronie Allegro widoczne są statusy zwrotu, ale danych tych nie można zobaczyć w API.

Zgłaszam sugestię, żeby te dane pojawiły się w response oraz dodanie filtra, który pozwalałby pobierać zwroty klienckie z konkretnym statusem.

**Comment by hahyes** [comment]
Podbijam temat, statusy powinny jak najbardziej być w danych zwrotu konsumenckiego.

**Comment by vendoremo** [comment]
Popieram, właśnie zabrałem się za integrację zwrotów. To jakiś koszmar. Po wszystkim można przefiltrować, tylko nie po tym, po czym potrzeba.

**Comment by paafcioc22** [comment]
również uważam że przydałby się filtr po statusie jak na web

**Comment by MartaNowaczyk** [comment]
UPDATE: Dzięki tej sugestii od kwietnia statusy [są dostępne w zwrotach klienckich](https://developer.allegro.pl/news/zwroty-klienckie-dodalismy-informacje-o-statusie-zwrotu-9gzwkO7ebuk). Zmieniłam status na "Suggestions - resolved".
